tests/data/fixtures/fasta_inputs/catnap_nt_light.fasta

```diff
@@ -271,11 +271,6 @@ GAGTGTTCACAGCAGATACTTTGCCTGGTACCAGCATAAACCTGGCCAGCCTCCCAGACTCCTCATCTATGGTGGGTCCA
 CCAGGGCCACTGGCATCCCTAATAGATTCAGTGCCGGCGGGTCTGGGACACAGTTCACTCTCACCGTCAACAGACTGGAG
 GCTGAAGATTTTGCGGTATATTACTGTCAGCAGTATGGTCGCTCCCCGTACACGTTCGGCCAAGGGACCAAGGTGGAGAT
 CAGAC
->CH103_Liao2013_CH103_light_chain_from_Liao2013,_Supplementary_Table_4
```

Comment:
This sequence shouldn't have passed, but will be great for the missing sequence needed to help with renumbering codecoverage